django/db/backends/dummy/base.py

```diff
@@ -76,6 +76,7 @@ def __init__(self, *args, **kwargs):
         super(DatabaseWrapper, self).__init__(*args, **kwargs)
 
         self.features = BaseDatabaseFeatures(self)
+        self.features.supports_transactions = False
```

Comment:
Might be a good alternative to explicitly check for dummy backend in features.supports_transactions() method?